group_vars/all

```diff
@@ -56,4 +56,8 @@ build_version: v1.20.0-alpha.0.1402+d39214ade1d60c
 # build_version: d39214ade1d60c
 
 ##### Calico Configurations #####
-calico_version: v3.24.1
+calico_version: v3.24.5
+pod_subnet: 172.20.0.0/16
```

Comment:
lets move this out of calico section

Reply:
Moved `pod_subnet` field to Kubernetes configuration section of the file.